.github/workflows/ci.yml

```diff
@@ -115,3 +125,5 @@ jobs:
         bundle config path vendor/bundle
         bundle install --jobs 4 --retry 3
         bundle exec rake coverage:report
+      env:
+        RAILS_VERSION: '~> 7.0.0'
```

Comment:
Same comment above

Reply:
And largely the same response as above.  :)

The coverage task doesn't have a matrix, so we can't use a matrix value.  When I left it free-floating - so it used Rails main - I got errors.  Given it's a hard-coded Ruby 2.7, it made sense to me to use the highest Rails version that supports Ruby 2.7.

Again, open to alternate suggestions.